Project-I-Social-Card/social-card/src/App.js

```diff
@@ -1,22 +1,17 @@
 import React from 'react';
+import HeaderContainer from './components/HeaderComponents/HeaderContainer';
+import CardContainer from './components/CardComponents/CardContainer';
+import Footer from './components/FooterComponents/Footer';
 import './App.css';
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Social Card!</h3>
-      <p>
-        Begin by exploring the `components` directory. You'll notice we have a
-        few files that we've already included in there to get you started right
-        away building components. You'll need to make sure you include your
-        components that you build in this file to watch your app come to life
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className='grid-container'>
+      <HeaderContainer />
+      <div className="card">
+        <CardContainer />
+      </div>
```

Comment:
think about how this will be rendered to the DOM and what the virtual html structure would look like: you have your CardContainer component wrapped in a div here, but everything inside CardContainer is already wrapped in another div. with that in mind, this wrapping div is unnecessary